### ISSUE TITLE
SSL error on kpm deploy

### ISSUE THREAD

**Issue by grantmiller11** [stackanetes/stackanetes#160]
Trying to do a deploy and it looks like it's complaining about the SSL cert?

kpm deploy kpm.sh/stackanetes/stackanetes --namespace openstack --variables stackanetes/parameters.yaml
usage: kpm [-h]
           {inspect,show,run-server,jsonnet,logout,plugins,helm,generate,pull,deploy,list,remove,delete-package,version,exec,push,new,login,config,channel}
           ...
kpm: error:
message: 'HTTPSConnectionPool(host=''kpm.sh'', port=443): Max retries exceeded with url: /cnr/api/v1/packages/stackanetes/stackanetes/default/kpm/pull?format=json (Caused by SSLError(SSLError(1, u''[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)''),))'

pip show kpm | grep Version
Version: 0.25.0

**Comment by makomatic** [comment]
Same here...
I suppose the project is dead, right?

**Comment by PiotrProkop** [comment]
@makomatic  Yeah :smile:, if you want actively maintained and developed Openstack on Kubernetes use: https://github.com/openstack/openstack-helm

**Comment by makomatic** [comment]
Already found it in the meantime, thx :)

**Comment by pawelsocha** [comment]
@PiotrProkop can you add info about helm into Readme? :P

**Comment by PiotrProkop** [comment]
Good idea, will do!

**Comment by PiotrProkop** [comment]
@makomatic I have used `openstack-helm` and it works as expected
